rules/sinks/third_parties/sdk/seven/csharp.yaml

```diff
@@ -0,0 +1,13 @@
+
+#  Sink rule for ThirdParty SDK
+#  The id follows a format : "ThirdParties.SDK.<THIRD_PARTY_ORGANISATION>.<SUB_ORGANISATION_IF_APPLICABLE>"
+
+sinks:
+
+  - id: ThirdParties.SDK.Seven
+    name: Seven
+    domains:
+      - "seven.io"
+    patterns:
+      - "(?i)(sms77-api).*"
```

Comment:
Is this pattern correct?

Reply:
Yes.
Refer: https://www.nuget.org/packages/sms77-api/

Reply:
```
using System;
using System.Threading.Tasks;
using Client = Sms77.Api.Client;

class Program
{
    static async Task Main()
    {
        var apiKey = Environment.GetEnvironmentVariable("SMS77_API_KEY");
        var client = new Client(apiKey);
        var balance = await client.Balance();
        Console.WriteLine($"Current account balance: {balance}");
    }
}
```

https://github.com/seven-io/net-client

I think pattern should be `(?i)(Sms77.Api.Client).*`

Reply:
I see, rare case where package and namespace aren't the same.

Reply:
Good catch, let me fix.